docs/tutorials/python.md

```diff
@@ -0,0 +1,273 @@
+
+```shell
+$ pixi list
+Package          Version       Build               Size       Kind   Source
+bzip2            1.0.8         h93a5062_5          119.5 KiB  conda  bzip2-1.0.8-h93a5062_5.conda
+ca-certificates  2024.2.2      hf0a4a13_0          152.1 KiB  conda  ca-certificates-2024.2.2-hf0a4a13_0.conda
+libexpat         2.6.2         hebf3989_0          62.2 KiB   conda  libexpat-2.6.2-hebf3989_0.conda
+libffi           3.4.2         h3422bc3_5          38.1 KiB   conda  libffi-3.4.2-h3422bc3_5.tar.bz2
+libsqlite        3.45.2        h091b4b1_0          806 KiB    conda  libsqlite-3.45.2-h091b4b1_0.conda
+libzlib          1.2.13        h53f4e23_5          47 KiB     conda  libzlib-1.2.13-h53f4e23_5.conda
+markdown_it_py   3.0.0                             224.2 KiB  pypi   markdown_it_py-3.0.0-py3-none-any.whl
+mdurl            0.1.2                             22.8 KiB   pypi   mdurl-0.1.2-py3-none-any.whl
+ncurses          6.4.20240210  h078ce10_0          801 KiB    conda  ncurses-6.4.20240210-h078ce10_0.conda
+openssl          3.2.1         h0d3ecfb_1          2.7 MiB    conda  openssl-3.2.1-h0d3ecfb_1.conda
+pixi_py          0.1.0                                        pypi   . (editable)
+pygments         2.17.2                            4.1 MiB    pypi   pygments-2.17.2-py3-none-any.whl
+python           3.12.2        hdf0ec26_0_cpython  12.5 MiB   conda  python-3.12.2-hdf0ec26_0_cpython.conda
+readline         8.2           h92ec313_1          244.5 KiB  conda  readline-8.2-h92ec313_1.conda
+rich             13.7.1                            925.4 KiB  pypi   rich-13.7.1-py3-none-any.whl
+tk               8.6.13        h5083fa2_1          3 MiB      conda  tk-8.6.13-h5083fa2_1.conda
+tzdata           2024a         h0c530f3_0          117 KiB    conda  tzdata-2024a-h0c530f3_0.conda
+xz               5.2.6         h57fd34a_0          230.2 KiB  conda  xz-5.2.6-h57fd34a_0.tar.bz2
+```
+
+Here you can see the different conda and pypi packages listed. And as you can see the `pixi_py` package, that we are working on, is installed in editable mode. Every environment in pixi is isolated, but re-uses files that are hard-linked from a central cache directory. This means that you can have multiple environments with the same packages, but only have the individual files stored once on disk.
+
+## Getting code to run
+
+Lets add some code to the `pixi_py` package. We will add a new function to the `pixi_py/__init__.py` file:
+
+```python
+from rich import print
+
+def hello():
+    return "Hello, [bold magenta]World[/bold magenta]!", ":vampire:"
+
+def say_hello():
+    print(*hello())
+```
+
+Let's see if it this works by running:
+
+```shell
+pixi r python -c "import pixi_py; pixi_py.say_hello()"
+Hello, World! 🧛
+```
+
+So we are running a conda installed python. And we are importing the `pixi_py` package that is installed in editable mode. And we are calling the `say_hello` function that we just added. And it works! Cool!
```

Comment:
maybe we can write something like:
So we are running a python which is bring by pixi from conda

Reply:
Changed the wording :)